example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js

```diff
@@ -104,11 +104,19 @@ describe('The Ingest Granule failure workflow', () => {
   });
 
   afterAll(async () => {
-    // clean up stack state added by test
-    await deleteGranule({
+    // The granule provided cannot clean up the
+    // fake S3 objects, so we expect it to fail
+    const granuleResponse = await deleteGranule({
       prefix: config.stackName,
       granuleId: inputPayload.granules[0].granuleId,
+      pRetryOptions: {
+        retries: 0,
+      },
+      expectedStatusCode: 400,
     });
+    if (JSON.parse(granuleResponse.body).code !== 'NoSuchBucket') {
```

Comment:
So the granule is deleted even when it throws 400? I see now that deleting the files from S3 happens outside the transaction. That seems like a deliberate choice that I'm not sure I want to revise now, but it is kind of odd

Reply:
Yeah, I'm not sure I like that much, but the refactor would entail alternate endpoint behavior in the form of a flag/an additional endpoint/etc.